entrypoint.sh

```diff
@@ -95,7 +95,7 @@ CommitChanges () {
     git add --force "$DESTINATIONDIR"
     
     # Commit all the changed to the the GitHub Pages branch
-    git commit -m "Auto commit" || exit 1
+    git commit -m "Auto commit" || exit 0
```

Comment:
Thanks for the PR but I do not agree with the fix. This fix also suppresses other errors from `git commit` which is not desired.
Can you first create an [issue](https://github.com/langroodi/doxygenize/issues) to indicate how to reproduce the situation? Because I guess if nothing has been changed in the Doxygen documentation, git commit won't fail.